exec/prepare-package/src/index.ts

```diff
@@ -62,3 +64,16 @@ function packageShouldBeBuilt (manifest: PackageManifest, pkgDir: string): boole
   const mainFile = manifest.main ?? 'index.js'
   return !fs.existsSync(path.join(pkgDir, mainFile))
 }
+
+function getJoinedPath (root: string, sub: string) {
```

Comment:
Let's rename it to safeJoin
```suggestion
function safeJoinPath (root: string, sub: string) {
```